### PR TITLE
parse set and set_with_title from 856

### PR DESCRIPTION
**Pull request by lmcglohon**
@ndushay This is ready for your review

### CODE REVIEW

File: stanford-sw/test/data/managedPurlTests.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
       <subfield code="x">DDR-PURL</subfield>
       <subfield code="x">item</subfield>
       <subfield code="x">image</subfield>
-      <subfield code="x">file:file1.jpg, file2.jpg</subfield>
```
Comment (ndushay) [inline stanford-sw/test/data/managedPurlTests.xml]: i'm kinda muzzy on all this, but it *seems* like a good idea to have at least one example with multiple files, whether here or inline data somewhere.

File: stanford-sw/test/src/edu/stanford/ManagedPurlTests.java [hunk 2]
```diff
@@ -32,46 +32,60 @@ public final void setup()
...
 		assertDocHasNoFieldValue("ManagedAnd2UnmanagedPurlCollection", fldName, "book");
 	}
-	
+
+/**
+ * Test method for fileId.
+ */
+@Test
+public final void testSetFileId() throws IOException, ParserConfigurationException, SAXException
+{
```
Comment (ndushay) [inline stanford-sw/test/src/edu/stanford/ManagedPurlTests.java]: most of the time I don't care much about alignment, but left justified for this feels uncomfy.

**Comment by ndushay** [comment]
a question and a quibble for you via line notes.

File: stanford-sw/test/data/managedPurlTests.xml (hunk 1 above)
Comment (lmcglohon) [inline stanford-sw/test/data/managedPurlTests.xml]: There will never be more than one file.

**Comment by lmcglohon** [comment]
@ndushay fixed the alignment but your note went away.

**Pull request merged**